lib/routes/index.js

```diff
@@ -38,21 +37,22 @@ module.exports = function routes(app, options, done) {
         });
       }
 
-      app.feedsme.change(req.params.env, req.body, function changed(err) {
-        if (err) {
-          app.log.error('failed to trigger a change for ' + req.body.name + ', received error: ' + err.message);
-
-          return res.status(500).send({
-            message: err.message,
-            ok: false
-          });
-        }
+      try {
+        await app.feedsme.change(req.params.env, req.body);
 
-        res.status(200).send({
+        return res.status(200).send({
```

Comment:
From an `async` and `await` standpoint, wouldn't you want this outside of the `try`? If the error occurs in `res.send` or `res.status` you will attempt to send headers twice which will likely crash the process.

Reply:
I see your point. I've just been trying to get the right sense of how to properly do this with async await